src/WebJobs.Script.WebHost/Management/MeshServiceClient.cs

```diff
@@ -142,8 +145,19 @@ private async Task<HttpResponseMessage> SendAsync(IEnumerable<KeyValuePair<strin
         {
             var operationName = formData.FirstOrDefault(f => string.Equals(f.Key, Operation)).Value;
             _logger.LogDebug($"Sending mesh request {operationName}");
-            var res = await _client.PostAsync(_environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI),
-                new FormUrlEncodedContent(formData));
+            //var res = await _client.PostAsync(_environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI),
+            //    new FormUrlEncodedContent(formData));
+            //var res = await _client.PostAsync(string requestUri, HttpContent content);
+
+            var request = new HttpRequestMessage(HttpMethod.Post, _environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI))
+            {
+                Content = new FormUrlEncodedContent(formData)
+            };
+
+            request.Headers.Add("fx-current-instance", _environment.GetEnvironmentVariable(EnvironmentSettingNames.ContainerName));
```

Comment:
I see we have a[ constant file for some other headers](https://github.com/Azure/azure-functions-host/blob/afafa87da101d8ed2fe1fa7aee317692db252786/src/WebJobs.Script/ScriptConstants.cs#L100-L107). Should we move this there? 

Also what is this "fx-" prefix stands for?

Reply:
Moved the header to ScriptConstants 👍